src/mobileJoystick.js

```diff
@@ -0,0 +1,96 @@
+// Create an SVG element
+const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+const circleInside = document.createElementNS('http://www.w3.org/2000/svg', 'circle');
+
+// Set the width as a percentage of the screen size
+const screenWidth = window.innerWidth;
+const svgWidth = (20 / 100) * screenWidth;
+const joystickRadius = svgWidth / 2;
+const joystickNippleRadius = svgWidth / 8;
+
+export const mobileVelocity = {
+    createdJoystick: false,
+    x: 0,
+    y: 0, 
+    left: false,
+    right: false,
+    up: false,
+    down: false,
+}
+
+export function createJoystick() {
+
+svg.setAttribute('width', svgWidth);
```

Comment:
The code inside the function should be indented by 2 spaces.

Reply:
- [x]